### PR TITLE
updating package in slides

### PR DESCRIPTION
**Pull request by carriewright11**

### CODE REVIEW

File: modules/Intro/Intro.Rmd [hunk 1]
```diff
@@ -397,7 +397,7 @@ You can just copy and paste the below code into your console - we'll explain wha
 ```{r, packageInstall, eval=FALSE}
 install.packages("remotes")
 library(remotes)
-install_github("muschellij2/jhur")
+remotes::install_github("jhudsl/jhur")
```
Comment (avahoffman) [inline modules/Intro/Intro.Rmd:400]: Shouldn't need `remotes::`

Comment (carriewright11) [reply modules/Intro/Intro.Rmd:400]: right! sorry I missed that- was trying to just replace the name :P and missed that when I tried to resolve the conflict

**Comment by carriewright11** [comment]
looks like it is upset about the other files that changed for the rendered files... maybe I will wait a bit and then merge main into here again

**Comment by avahoffman** [comment]
no prob! Sometimes it helps to delete the preview branch and just let actions re-create it.

**Pull request merged**